src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
Let`s use `<img>` to improve SEO. `<img>` allows to add an `alt `attribute, which will be used by search engine to understand what is on a picture, and alt text will be shown to user if there was some problems with loading the image